homeassistant/components/config/scene.py

```diff
@@ -5,7 +5,10 @@
 from typing import Any
 import uuid
 
-from homeassistant.components.scene import DOMAIN, PLATFORM_SCHEMA
+from homeassistant.components.scene import (
+    DOMAIN,
+    PLATFORM_SCHEMA as SCENE_PLATFORM_SCHEMA,
+)
```

Comment:
Actually this one isn't being extended so maybe we need to keep ?

Reply:
Oh I was to fast on the merge there...

But looking at it, I kind like it for the consistency?

Reply:
Oh! I missed this comment.
@frenck what do you think?

Reply:
I'm really in the middle... I have no strong opinion on way or the other.